src/stacks-hierarchy/Root_TicketAssistantStack/Root_TicketAssistantStack.tsx

```diff
@@ -0,0 +1,92 @@
+import {TicketAssistant_FrequencyScreen} from '@atb/stacks-hierarchy/Root_TicketAssistantStack/TicketAssistant_FrequencyScreen';
+import {TicketAssistant_CategoryPickerScreen} from '@atb/stacks-hierarchy/Root_TicketAssistantStack/TicketAssistant_CategoryPickerScreen';
+import {TicketAssistant_DurationScreen} from '@atb/stacks-hierarchy/Root_TicketAssistantStack/TicketAssistant_DurationScreen';
+import {TicketAssistant_ZonePickerScreen} from '@atb/stacks-hierarchy/Root_TicketAssistantStack/TicketAssistant_ZonePickerScreen';
+import {TicketAssistant_SummaryScreen} from '@atb/stacks-hierarchy/Root_TicketAssistantStack/TicketAssistant_SummaryScreen';
+import {FullScreenHeader} from '@atb/components/screen-header';
+import {RootStackScreenProps} from '@atb/stacks-hierarchy';
+
+const Tab = createMaterialTopTabNavigator<TicketAssistantStackParams>();
+const themeColor: StaticColorByType<'background'> = 'background_accent_0';
+
+type Props = RootStackScreenProps<'Root_TicketAssistantStack'>;
+
+export const Root_TicketAssistantStack = ({navigation}: Props) => {
+  const styles = useStyles();
+  const [activeTab, setActiveTab] = useState(0);
+  const [previousTab, setPreviousTab] = useState<any>();
+  return (
+    <>
+      {activeTab !== 0 ? (
+        <FullScreenHeader
+          leftButton={{
+            type: 'back',
+            //Navigate to previous tab
+            onPress: () => {
+              navigation.navigate(previousTab);
+            },
+          }}
+          rightButton={{type: 'close'}}
+        />
+      ) : (
+        <FullScreenHeader rightButton={{type: 'close'}} />
+      )}
+      <SafeAreaView style={styles.container}>
+        <Tab.Navigator
+          tabBar={(props: MaterialTopTabBarProps) => {
+            setActiveTab(props.state.index);
+            setPreviousTab(props.state.routes[props.state.index - 1]);
+            return <PageIndicator {...props} />;
+          }}
+          tabBarPosition="bottom"
+          initialRouteName="TicketAssistant_WelcomeScreen"
+        >
+          <Tab.Screen
+            name="TicketAssistant_WelcomeScreen"
+            component={TicketAssistant_WelcomeScreen}
+          />
+          <Tab.Screen
+            name="TicketAssistant_FrequencyScreen"
+            component={TicketAssistant_FrequencyScreen}
+          />
+          <Tab.Screen
+            name="TicketAssistant_CategoryPickerScreen"
+            component={TicketAssistant_CategoryPickerScreen}
+          />
+          <Tab.Screen
+            name="TicketAssistant_DurationScreen"
+            component={TicketAssistant_DurationScreen}
+          />
+          <Tab.Screen
+            name="TicketAssistant_ZonePickerScreen"
+            component={TicketAssistant_ZonePickerScreen}
+          />
+          <Tab.Screen
+            name="TicketAssistant_SummaryScreen"
+            component={TicketAssistant_SummaryScreen}
+          />
+        </Tab.Navigator>
+      </SafeAreaView>
+    </>
+  );
+};
+
+// @ts-ignore
```

Comment:
Seems like this might be not needed, at least I didn't get any issues when linting and compiling TS.